proof/infoflow/ADT_IF.thy

```diff
@@ -1379,11 +1405,17 @@ lemma check_active_irq_if_wp:
    \<lbrace>P\<rbrace>"
   by (wpsimp wp: dmo_getActiveIRQ_wp simp: check_active_irq_if_def)
 
+
+context ADT_IF_1 begin
+
 lemma handle_preemption_if_only_timer_irq_inv[wp]:
   "handle_preemption_if tc \<lbrace>only_timer_irq_inv irq st\<rbrace>"
   by (wp only_timer_irq_inv_pres handle_preemption_if_irq_masks handle_preemption_if_domain_sep_inv
       | simp | blast)+
 
+end
+
+
```

Comment:
Very minor but you added these extra newlines a couple of times around context blocks like this. Was that intentional? I don't think we usually do it, unless the block is its own section in some way.

Reply:
The current security proof style is to treat context blocks as sections, so will keep as is for now